src/ScriptCs.CSharp/ScriptCs.CSharp.csproj

```diff
@@ -38,22 +38,22 @@
       <HintPath>..\packages\Common.Logging.2.1.2\lib\net40\Common.Logging.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.CodeAnalysis">
-      <HintPath>..\..\packages\Microsoft.CodeAnalysis.Common.1.0.0-rc2-20150306-06\lib\net45\Microsoft.CodeAnalysis.dll</HintPath>
+      <HintPath>..\packages\Microsoft.CodeAnalysis.Common.1.0.0-rc2-20150306-06\lib\net45\Microsoft.CodeAnalysis.dll</HintPath>
```

Comment:
Yes, they are intentional in this PR. I don't know why they are currently pointing one level up, since  that causes compilation to fail, as that is not where the DLL's are.